Jonathan-jonathankern/Homework-2/temp-converter/js/main.js

```diff
@@ -0,0 +1,111 @@
+/*
+    1. Create a function that accepts a temperature value in Fahrenheit and converts it to Celsius.
+      Note: Equation to convert F to C:
+      C = (F - 32) / 1.8
+      Test your function.
+    2. Create a second function that accepts a temperature value in Celsius and converts it to Fahrenheit.
+      Note: Equation to convert C to F:
+      F = 1.8 * C + 32
+      Test your function.
+
+    BONUS 1: Update your functions to use the value entered in the text box on the web page, and to perform calculations only when the appropriate button is clicked:
+      1.  Set the value of the temp variable to the value of the element with the id value 'temperature'.
+          (Hint: read about the document.querySelector() method.)
+      2.  Set the appropriate function to run when each button is clicked.
+        (Hint: read about the addEventListener() method.)
+
+    BONUS 2: Display the result of each calculation in the browser window rather than in the console:
+      1.  At the end of each function, set the text of the element with the id value 'result' to the same text string you were previously logging to the console.
+
+    BONUS 3: Clear the contents of the text box after you've displayed your results, so a user can easily enter another temperature to convert. (Hint: Read about the 'value' property in JavaScript. What string value could you assign as the value of an input box that would give it no content to display?)
+*/
+
+// assign #fahrenheit_to_celsius variable
+let fahrenheitToCelsius = document.getElementById('fahrenheit_to_celsius');
+
+// assign #celsius_to_fahrenheit variable
+let celsiusToFahrenheit = document.getElementById('celsius_to_fahrenheit');
+
+// assign #celsius_to_fahrenheit variable
+let clearContents = document.getElementById('clear_contents');
+
+let showResult = document.getElementById('result');
+
+// add click event listener to fahrenheitToCelsius
+let fahrenheitToCelsiusEventHandler = fahrenheitToCelsius.addEventListener("click", function() {
+    // console.log("fahrenheit was clicked");
+
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value;
+
+    fahrenheitToCelsiusConverter(temperature);
+});
+
+// add click event listener to celsiusToFahrenheit
+let celsiusToFahrenheitEventHandler = celsiusToFahrenheit.addEventListener("click", function() {
+    // console.log("celsius was clicked");
+
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value;
+
+    celsiusToFahrenheitConverter(temperature);
+});
+
+// add click event listener to clearContents
+let clearContentsEventHandler = clearContents.addEventListener("click", function() {
+    // console.log("clearContents was clicked");
+    // store temperature from input field
+    let temperature = document.querySelector('#temperature').value = ' ';
```

Comment:
Notice that you have 2 assignment operators in this statement, which is a practice you want to avoid. Instead, it should be 2 statements -- one to assign the value to the temperature variable, and one to zero out the value in the box. Also, you're assigning the new value as a single space here, but it's best practice to use an empty string instead ('' without a space between); this ensures there's no value at all in the box, so if there's placeholder text, it will be displayed. (A space is counted as content.)